scripts/iam_policy.json

```diff
@@ -24,7 +24,8 @@
           "states:*",
           "backup:*",
           "backup-storage:MountCapsule",
-          "ssm:GetParameter"
+          "ssm:GetParameter",
+          "ssm:GetParameters"
```

Comment:
The IAM user that is created here is used to deploy FWoA to AWS. This IAM user can also be used during development to run `serverless offline`. If the IAM User is being used during development, the user should also have `ssm:GetParameters` permission.

This info is referenced here.
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/api/DEVELOPMENT.md#development

Reply:
I don't agree with this. Permissions to deploy and permissions needed at runtime are a very different thing.

This file is already a bit of a mess (what does  `iam_policy.json` means? only the docs tell you that this is meant for deployment, we should probably rename it. Also, It has too many `*` permissions). I don't think that adding runtime permissions on top of it is a good idea.

When running sls offline you can either:
- Assume an Admin role. It's ok, you are a dev testing stuff in a dev AWS account. You can use an admin role to not get any permission issues and just focus on development.
- Assume a role that matches the permissions of the lambda function role that you are testing. I **think** that serverless has something to help set this up(when testing individual lambda functions). Running it this way is closer to how the lambda runs once deployed on AWS and can be helpful to test permissions.

Reply:
I like the idea of AdminitrativeAccess. That's actually how I currently run the code locally too. I'll revert the changes here, and update the text in DEVELOPMENT.md